src/diffusers/models/transformers/transformer_sd3.py

```diff
@@ -231,9 +231,16 @@ def fuse_qkv_projections(self):
 
         self.original_attn_processors = self.attn_processors
 
-        for module in self.modules():
```

Comment:
is this needed? `self.modules` will get all the attention models, nested or not

Reply:
You are 100 percent right. I have fixed it in d563b9eac.